Google.PowerShell/BigQuery/BqTabledata.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections.Generic;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Instantiates a new BQ schema or adds a field to a pre-existing schema.
+    /// </para>
+    /// <para type="description">
+    /// If no existing schema is passed in, this command will create a new schema with one column. 
+    /// If an existing schema is supplied, this command will add a new column to that schema. 
+    /// Required fields for each column are Name and Type. Possible values for Type include STRING, 
+    /// BYTES, INTEGER (also called INT64), FLOAT (also FLOAT64), BOOLEAN (also BOOL), TIMESTAMP, 
+    /// DATE, TIME, DATETIME, and RECORD (where RECORD indicates that the field contains a nested 
+    /// schema. Also called STRUCT). Possible values for Mode include NULLABLE, REQUIRED, and 
+    /// REPEATED. Case is ignored for both Type and Mode. This command returns the new or modified 
+    /// TableSchema object.
+    /// </para>
+    /// <example>
+    ///   <code>
+    /// PS C:\> $table = Get-BqDataset “book_data” | Get-BqTable "21st_century"
+    /// PS C:\> $schema = New-BqSchema -Name "Title" -Type "STRING" -Description "Book Title"
+    /// PS C:\> $schema = $schema | New-BqSchema -Name "Author" -Type "STRING" -Description "Book Author"
+    /// PS C:\> $table.Schema = $schema
+    /// PS C:\> $table | Set-BqTable
+    ///   </code>
+    ///   <para>This will create a new schema, assign it to a table, and then send the 
+    ///   revised table to the server to be saved.</para>
+    /// </example> 
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/tables)">
+    /// [BigQuery Tables]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "BqSchema")]
+    public class NewBqSchema : PSCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// The existing TableSchema that you wish to add a column to. 
+        /// If this value is not present, a new schema will be created.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipeline = true)]
+        public TableSchema InputObject { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The name of the column to be added. The name must be unique among columns in each schema.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = true)]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// <para type="description">
+        /// The type of the column to be added. Possible values include STRING, BYTES, INTEGER (INT64), 
+        /// FLOAT (FLOAT64), BOOLEAN (BOOL), TIMESTAMP, DATE, TIME, DATETIME, and RECORD (STRUCT).
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = true)]
+        [ValidateSet("STRING", "BYTES", "INTEGER", "INT64", "FLOAT", "FLOAT64", "BOOLEAN", "BOOL", 
```

Comment:
Yep.  Was having issues with the parameter tags and forgot to put that back in after testing.